app/components/ExternalIps.tsx

```diff
@@ -6,13 +6,21 @@
  * Copyright Oxide Computer Company
  */
 
-import { useApiQuery } from '@oxide/api'
+import { Link } from 'react-router'
+
+import { useApiQuery, type ExternalIp } from '@oxide/api'
 
 import { EmptyCell, SkeletonCell } from '~/table/cells/EmptyCell'
 import { CopyableIp } from '~/ui/lib/CopyableIp'
+import { Slash } from '~/ui/lib/Slash'
 import { intersperse } from '~/util/array'
+import { pb } from '~/util/path-builder'
 import type * as PP from '~/util/path-params'
 
+/** Move ephemeral IP (if present) to the end of the list of external IPs */
+export const orderIps = (ips: ExternalIp[]) =>
+  ips.sort((a) => (a.kind === 'ephemeral' ? 1 : -1))
```

Comment:
`import * as R from 'remeda'` and use [`R.sortBy`](https://remedajs.com/docs/#sortBy) to avoid the mutation (like the comment said). I think this mutates what's in the query cache, which freaks me out.